### DISCUSSION TITLE
Is this a regression in grace note rendering?

### DISCUSSION THREAD

**Discussion by lannka** [CoderLine/alphaTab#603] [Q&A]
**In the latest code, I'm getting grace note rendered like this:**

<img width="461" alt="Screen Shot 2021-05-21 at 5 14 41 PM" src="https://user-images.githubusercontent.com/8496897/119208701-39877880-ba58-11eb-82ef-995ace24bba2.png">

**Comparing to the release 1.2, it's quite different:**
<img width="328" alt="Screen Shot 2021-05-21 at 5 17 14 PM" src="https://user-images.githubusercontent.com/8496897/119208760-6b004400-ba58-11eb-9937-2258ca0d2f54.png">

**Comment by Danielku15** [comment]
Tested alphaTab with 1.3.0-alpha.119 and the note flag is rendered for me. Same when testing the develop branch locally. The paddings seem a bit suboptimal in this scenario but in general the display is correct. I will take it up to have a bit a padding before grace notes. 
https://jsfiddle.net/danielku15/hnv87ugt/

![image](https://user-images.githubusercontent.com/674916/119236850-e259d480-bb39-11eb-9ebf-75b55397d42f.png)

So I would say: there is no regression in the code in general. But maybe there is something special on your setup. Can you please try to file a full [bug report](https://github.com/CoderLine/alphaTab/issues/new/choose) and provide all details **including** a Fiddle showing the issue. This might either show you where something is wrong in your setup, or it will show me where exactly there might be a bug.

**Reply by lannka** [reply]
Really weird that it's just on my local dev mode. It's not a regression as I can reproduce on a much older commit. I can reproduce even on a clean clone of the repo, with the default settings that the `alphatex-editor.html` has.

I might need more time to figure out what's happening.
